roles/ovirt-cluster-upgrade/README.md

```diff
@@ -16,7 +16,7 @@ Role Variables
 |-------------------------|-----------------------|-----------------------------------------------------|
 | cluster_name            | Default               | Name of the cluster to be upgraded.                 |
 | stopped_vms             | UNDEF                 | List of virtual machines to stop before upgrading.      |
-| stop_pinned_to_host_vms | false                 | Specify whether to stop virtual machines pinned to the host being upgraded. If true, the pinned virtual machines will be stopped.     |
+| stop_pinned_to_host_vms <br/> <i>alias: stop_non_migratable_vms</i>  | false                 | Specify whether to stop virtual machines pinned to the host being upgraded. If true, the pinned non-migratable virtual machines will be stopped and host will be upgraded, otherwise the host will be skipped. |
```

Comment:
Shouldn't we have it in a opposite way? I mean stop_non_migratable_vms as correct parameter and stop_pinned_to_host_vms as an deprecated alias? If so, then I would also change the code to not use stop_pinned_to_host_vms at all (expect initialization of stop_non_migratable_vms).

Reply:
Sure, will do.